web/thesauruses/objectivec/2/queues_stacks.json

```diff
@@ -0,0 +1,223 @@
+{
+  "meta": {
+    "language": "objectivec",
+    "language_version": "2.0",
+    "language_name": "Objective-C",
+    "structure": "queues_stacks",
```

Comment:
Oops, there's an extra comma here. (Trailing commas are often correct for Python, JS, etc. but not for JSON.)

Reply:
my goodness. That's what I get for eating the meta after testing. I removed the extra comma.